lawnchair/src/app/lawnchair/allapps/SearchItemBackground.kt

```diff
@@ -18,10 +18,10 @@ class SearchItemBackground(
     private val resources = context.resources
 
     private val searchDecorationPadding = resources.getDimensionPixelSize(R.dimen.search_decoration_padding)
-    private val focusHighlight = ColorTokens.FocusHighlight.resolveColor(context)
-    private val groupHighlight = if (showBackground) ColorTokens.GroupHighlight.resolveColor(context) else 0
+    val focusHighlight = ColorTokens.FocusHighlight.resolveColor(context)
+    val groupHighlight = if (showBackground) ColorTokens.GroupHighlight.resolveColor(context) else 0
 
-    private val cornerRadii: FloatArray
+    val cornerRadii: FloatArray
```

Comment:
Move these public fields under private ones.

Reply:
this is intended as we used it in `SearchAdaptedItem` 
![image](https://github.com/LawnchairLauncher/lawnchair/assets/36076410/c6212af0-061e-49e6-aa33-bc046442d2f1)

Reply:
Yeah, I mean't public fileds should be placed below private fields